p2p/src/peer_manager/peerdb/mod.rs

```diff
@@ -93,6 +92,11 @@ impl<S: PeerDbStorage> PeerDb<S> {
             anchor_addresses,
         } = LoadedStorage::load_storage(&storage)?;
 
+        let banned_addresses = banned_addresses
+            .into_iter()
+            .map(|(addr, dur)| (addr, Time::from_duration_since_epoch(dur)))
+            .collect::<BTreeMap<BannableAddress, Time>>();
+
```

Comment:
It'd be nicer if `LoadedStorage` used Time already,

Reply:
Done